app/services/open_library_service.rb

```diff
@@ -0,0 +1,14 @@
+class OpenLibraryService
+  include HTTParty
+
+  def book(isbn)
+    params_url = "bibkeys=ISBN:#{isbn}&format=json&jscmd=data"
+    HTTParty.get(base_uri + params_url)
```

Comment:
So the controller return the response of HTTParty? Can we use a private method to format the response as we want?